scripts/helpers.sh

```diff
@@ -62,3 +62,46 @@ command_exists() {
   local command="$1"
   command -v "$command" &> /dev/null
 }
+
+get_tmp_dir() {
+  local tmpdir="${TMPDIR:-${TMP:-${TEMP:-/tmp}}}"
```

Comment:
From `man tmux`
> tmux stores the server socket in a directory under TMUX_TMPDIR or /tmp if it is unset.

So either `TMUX_TMPDIR` or `/tmp` should exist

Reply:
Maybe that's why `tmux-copycat` does that if inside tmux env `TMUX_TMPDIR` turn into `TMPDIR`

Reply:
Sorry for the back and forth

Reply:
maybe; though I don't really see much point in reverting - surely checking TMP & TEMP is fine?

Reply:
I'm ok with both solution